scss/backend/dot-admin/portlets/_add-relationship.scss

```diff
@@ -0,0 +1,21 @@
+.add-relationship {
+    .add-relationship__form {
```

Comment:
You don't need to nest ```.add-relationship__form``` remember specifity with BEM it's done by the class name.